### ISSUE TITLE
Diorite

### ISSUE THREAD

**Issue by Chappy130** [Thyriel/Minecraft-AEONs#5]
Where do you find Diorite? It is required for one of the builder hut setups and we have been unable to find any. any help is appreciated.

**Comment by Thyriel** [comment]
Same as in vanilla minecraft:
https://minecraft.gamepedia.com/Diorite

**Comment by Cyprex** [comment]
no, quark changes the way diorite, granite and andesite spawns to huge infrequent, biome-dependent clusters, and this is enabled in the config of Aeons 1.4.
https://quark.vazkii.us/img/stone-table.jpg
https://quark.vazkii.us/ world module, stone revamp section
However, all the rocks are craftable if you can't find them

**Comment by Cyprex** [comment]
For the record, in the beneath dimension Granite, Diorite and Andesite spawn commonly in small clusters.
